nir/data_ir/graph.py

```diff
@@ -0,0 +1,261 @@
+from __future__ import annotations
+from dataclasses import dataclass
+from enum import Enum
+from typing import Dict, Union
+import numpy as np
+from nir import NIRGraph
+
+
+class Interpolation(Enum):
+    NONE = "None"
```

Comment:
Why should this be an Enum? Do we expect other kinds of interpolation methods in the future?